internal/logs/aggregate/traefik.go

```
@@ -264,49 +391,242 @@ func aggregateTraefikJSON(rows []traefikJSON, since, now time.Time, rng models.D
+	return r.getInt64("Duration", "duration")
+}
+
+func (r traefikJSON) getRouter() string {
+	return r.getStr("RouterName", "router")
+}
+
+func (r traefikJSON) getService() string {
+	return r.getStr("ServiceName", "service")
+}
+
+func (r traefikJSON) getServiceAddr() string {
```

golangci-lint (2.12.2)

[error] func traefikJSON.getServiceAddr is unused

(unused)